crates/hir-ty/src/infer.rs

```diff
@@ -40,6 +40,7 @@ use la_arena::{ArenaMap, Entry};
 use rustc_hash::{FxHashMap, FxHashSet};
 use stdx::{always, never};
 
+use crate::TraitEnvironment;
 use crate::{
```

Comment:
huh, did r-a fail at merging this into the use tree here?

Reply:
Yes, when I remove it and use assist, it will add it here again.